packages/waci-pex/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@sphereon/waci-pex",
+  "version": "0.0.1",
+  "description": "Veramo SSI QR Code provider",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "ISsiQrCodeProvider": "./src/types/ISsiQrCodeProvider.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build"
+  },
+  "dependencies": {
+    "@sphereon/did-auth-siop": "^0.2.3",
+    "@veramo/core": "3.1.1-next.26",
+    "cross-fetch": "^3.1.4",
+    "react": "^17.0.2",
+    "react-qr-code": "^2.0.2",
```

Comment:
2.0.3 is latest version

Reply:
It would have downloaded this version automatically as the ^ was used there. But updated